internal/types/types.go

```diff
@@ -59,7 +59,7 @@ import (
 )
 
 // MaxDocumentLen is the maximum BSON object size.
-const MaxDocumentLen = 16777216
+const MaxDocumentLen = 16 * 1024 * 1024 // 16777216
```

Comment:
Maybe something like

```suggestion
const MaxDocumentLen = 16 * 1024 * 1024 // 16 Mb = 16777216 bytes
```